Layer 2/MultiLicenseV1.0.json

```diff
@@ -714,6 +731,86 @@
         "id": "vidm1pqmlp4od"
       }
     },
+    "CompactPrivileges ": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "properties": {
+        "JurisdictionAbbreviation": {
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "Jurisdiction": {
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "HomeJurisdictionAbbreviation": {
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "HomeJurisdiction": {
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "IssueDate": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "ExpirationDate": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "RenewalDate": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "Status": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "enum": [
+            "Active",
+            "Expired",
+            "Terminated"
+          ]
+        },
+        "Number": {
+          "type": [
+            "string",
+            "null"
+          ]
+      },
```

Comment:
looks like a closing } is missing here

Reply:
I don't think so. And stoplight's validation does not allow me to end one. The closing } is in row 813.